src/App.js

```diff
@@ -1,10 +1,50 @@
-import React from 'react';
+import {useState, useEffect} from 'react';
 import './App.css';
+import Ditto from './Ditto'
+import axios from "axios";
+
 
 function App() {
+    const [pokenames, setPokenames] = useState(null);
```

Comment:
Als dit altijd een array is, dan zou ik ook een lege array als default waarde gebruiken i.p.v. `null`